spec/features/admin_edits_user_profile_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+feature 'admin edits user profile', %Q{
+  As an admin, I want to view a list of all users, so that I can edit their
+  profiles.
+  Acceptance Criteria:
+  * [X] - I can edit the profile information of any user, including their
+          username and rank.
+  } do
+
+  let(:admin) do
+    FactoryGirl.create(:user, admin: true)
+  end
+
+  let(:user) do
+    FactoryGirl.create(:user)
+  end
+
+  scenario 'admin edits user profile' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.